src/runtime/pack_args.h

```diff
@@ -40,13 +40,24 @@ namespace tvm {
 namespace runtime {
 /*!
  * \brief argument union type of 32bit.
- * Choose 32 bit because most GPU API do not work well with 64 bit.
  */
 union ArgUnion {
```

Comment:
Is there a reason we duplicated this union?

Reply:
This supports 32 bit only use case. Not sure if we can replace this with 64 bit union.

Reply:
Let us rename to ArgUnion32 to avoid confusion

Reply:
done